packages/tables/src/Actions/CreateAction.php

```diff
@@ -58,14 +58,13 @@ protected function setUp(): void
                     $data = Arr::except($data, $pivotColumns);
                 }
 
-                $record = $relationship->create($data);
-
+                $pivotData = [];
```

Comment:
This line does not need to be here as the variable is not used outside the if statement, we can combine both if statements together